website/addons/onedrive/views.py

```diff
@@ -0,0 +1,191 @@
+"""Views for the node settings page."""
```

Comment:
Might not have been a blocker in December, but now that every addon that can use `generic_views` does, it is. Sorry that this PR went without much attention for so long.